plugins/woocommerce/changelog/add-51450-payment-providers-suggestions

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Handle payment provider action items
```

Comment:
No need for changelog entry for this one, since it's part of the NOX - right? 🙂

Reply:
yes - forgot to remove 🙈

Reply:
Done!